src/browser/browser.ts

```
@@ -0,0 +1,79 @@
+import { BrowserContext, chromium } from '@playwright/test'
+
+export async function openBrowser(
+    slowMo: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)